src/main/resources/db/mysql/initDB.sql

```diff
@@ -59,3 +59,16 @@ CREATE TABLE IF NOT EXISTS visits (
   description VARCHAR(255),
   FOREIGN KEY (pet_id) REFERENCES pets(id)
 ) engine=InnoDB;
+
+/*Code added by Maria Carolina Avila for the APPT team*/
+CREATE TABLE IF NOT EXISTS vet_schedule (
+    vet_ID INT(4) NOT NULL PRIMARY KEY,
+    room_ID VARCHAR(5) NOT NULL,
+    day_available VARCHAR(10),
+    visit_id INT(4),
+
+    FOREIGN KEY(vet_ID) REFERENCES vets(id),
+    FOREIGN KEY(visit_id) REFERENCES  visits(id)
+) engine=InnoDB;
+/*Note. Use the DAYNAME() function to convert visit_date to the corresponding day.
```

Comment:
I feel like it would be better to use `WEEKDAY()`, store `day_available` as an `INT`, and also insert a constraint that keeps that in the range of 0 to 6.